google/cloud/spanner/instance_admin_connection.cc

```diff
@@ -62,38 +62,59 @@ class InstanceAdminConnectionImpl : public InstanceAdminConnection {
 
   future<StatusOr<gcsa::Instance>> CreateInstance(
       CreateInstanceParams p) override {
-    auto operation = RetryLoop(
-        retry_policy_prototype_->clone(), backoff_policy_prototype_->clone(),
-        Idempotency::kNonIdempotent,
-        [this](grpc::ClientContext& context,
-               gcsa::CreateInstanceRequest const& request) {
-          return stub_->CreateInstance(context, request);
+    auto stub = stub_;
+    return google::cloud::internal::AsyncLongRunningOperation<gcsa::Instance>(
+        background_threads_->cq(), p.request,
+        [stub](
+            google::cloud::CompletionQueue& cq,
+            std::unique_ptr<grpc::ClientContext> context,
+            google::spanner::admin::instance::v1::CreateInstanceRequest const&
```

Comment:
here and below, consider: s/google::spanner::admin::instance::v1/gcsa/ ?

Reply:
Done.

Reply:
"and below"?

Reply:
I thought I got them both, no?

Reply:
I must be blind, sorry.